packages/@okta/vuepress-site/guides/sign-into-web-app/define-callback/springboot/define-route.md

```diff
@@ -0,0 +1,3 @@
+The Okta Spring Boot SDK will configure and host this route for you. By default, the route is hosted at `/authorization-code/callback`.
```

Comment:
The Okta Spring Boot SDK configures and hosts this route for you. By default, the route is hosted at `/authorization-code/callback`.